templates/README.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Accuracy_metric
+datasets: - some_dataset
+tags: - metric
```

Comment:
I think it's easier to read this way ?
```suggestion
datasets:
- some_dataset
tags:
- metric
```
also why the `datasets` field ?

Reply:
Sometimes metrics are limited to specific datasets, no?

Reply:
yup indeed, thanks ! Would be nice to have a comment that says this maybe

Reply:
Since some metrics are designed around one dataset (e.g. `glue`) we could indicate that here.